### PR TITLE
{lib,tools,vis}[GCCcore/5.4.0] gettext v0.19.8, libxml2 v2.9.4, XZ v5.2.2

### PR DESCRIPTION
**Pull request by vanzod**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/x/XZ/XZ-5.2.2-GCCcore-5.4.0.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+name = 'XZ'
+version = '5.2.2'
+
+homepage = 'http://tukaani.org/xz/'
+description = "xz: XZ utilities"
+
+toolchain = {'name': 'GCCcore', 'version': '5.4.0'}
+
+sources = [SOURCELOWER_TAR_BZ2]
+source_urls = ['http://tukaani.org/xz/']
```
Comment (boegel) [inline easybuild/easyconfigs/x/XZ/XZ-5.2.2-GCCcore-5.4.0.eb]: @vanzod please include the patch to avoid breaking `rpm`, see #4179

**Comment by vanzod** [comment]
Test report by @vanzod
**FAILED**
Build succeeded for 2 out of 3 (3 easyconfigs in this PR)
cermis - Linux debian 8.8, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.9
See https://gist.github.com/d0c54695fb3b3c639870252f189f6396 for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2415.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/892be645952c05028f3c1238884d22b3 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2032.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/488d29957a7b60f463ea5f18f56ad0c6 for a full test report.

**Comment by boegel** [comment]
@vanzod Any ideas what went wrong on your end?

**Comment by vanzod** [comment]
No idea. `XZ` compiles correctly and `EB` returns no errors on the stdout.
I have rebuilt `XZ` with debug messages but still I cannot see why it is reported as failed. I attached the log in case you want to look at it.
[debug_log.txt](https://github.com/hpcugent/easybuild-easyconfigs/files/1045751/debug_log.txt)

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
cermis - Linux debian 8.8, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.9
See https://gist.github.com/f3c0a6c9b65a9107cc7d18ceae8505af for a full test report.

**Comment by boegel** [comment]
Going in, thanks @vanzod!

**Pull request merged**